packages/finicity-adapter/src/adapter.ts

```diff
@@ -225,8 +226,13 @@ export class FinicityAdapter implements WidgetAdapter {
       case "credentialsUpdated":
         connection.status = ConnectionStatus.CONNECTED;
         break;
+      case "done":
+        return connection;
       default:
         switch (reason) {
+          case "complete":
+            connection.status = ConnectionStatus.CONNECTED;
+            break;
```

Comment:
This change fixed a problem where the redirect landing page or oauth complete page was saying "Something went wrong" even though it was successful because we need to pass a successful connection status back in the connection response when the complete event type occurs.